src/modules/limitOrders/updaters/MarketPriceUpdater/index.tsx

```diff
@@ -1,7 +1,45 @@
-import { useFetchMarketPrice } from './hooks/useFetchMarketPrice'
+import { useMemo } from 'react'
+import { useQuote } from 'modules/tradeQuote'
+import { useDerivedTradeState } from 'modules/trade/hooks/useDerivedTradeState'
+import { CurrencyAmount, Percent, Price } from '@uniswap/sdk-core'
+import { FractionUtils } from 'utils/fractionUtils'
+import { updateLimitRateAtom } from 'modules/limitOrders/state/limitRateAtom'
+import { useUpdateAtom } from 'jotai/utils'
+
+export const LIMIT_ORDERS_PRICE_SLIPPAGE = new Percent(1, 10) // 0.1%
 
 export function MarketPriceUpdater() {
```

Comment:
We now only need this code for market price, that will update the market rate based on the changes in quote.